### ISSUE TITLE
10.0.22621.3958 (unable to use RDP or reinstall the program)

### ISSUE THREAD

**Issue by 4Hillz** [stascorp/rdpwrap#3241]
![image](https://github.com/user-attachments/assets/8eb58794-c3c4-40bf-be2a-5cee0b89e384)

I did a new Windows update for Windows 11 Home and now my RDP is not working and was stuck on this screen. I tried to refresh it by deleted the RDP Wrapper in the programs file and then re-adding it, which results in the rdpwrapper.dll not having its encryption anymore, which may have done something. Whenever I try to reinstall it does not work because it says that RDP Wrapper is installed, but when I uninstall, it comes up with this, saying that it failed to set up TermService. 

![image](https://github.com/user-attachments/assets/5d83008a-4672-4b77-b123-952914abc41d)

**Comment by binarymaster** [comment]
Duplicate of #1112.

**Comment by 4Hillz** [comment]
I did that, and now I have a new problem. It is saying that the listener state is "not listening" for some reason even though I just reinstalled it.
![image](https://github.com/user-attachments/assets/444ca908-37ea-4ed1-8fd0-b82888d26d25)

I then went into the RDP Wrapper folder in Program Files and replaces the .ini file with the one I have attached to this document, as it was the one I was originally using when my RDP Wrapper worked before the Windows update. It still had the same issue saying that I am "not listening"
